job/job_test.go

```diff
@@ -173,3 +175,46 @@ func TestSetupFails(t *testing.T) {
 	// should still be able to log, see #_incident-130
 	j.With("happyThoughts", true).Error(err)
 }
+
+func TestDiscoverActiveWorkflowForMacFound(t *testing.T) {
+	var hwID string = "0eba0bf8-fake-4b4a-ab9f-6ebe93b90a94"
+	var wcl tw.WorkflowContextList = tw.WorkflowContextList{
+		WorkflowContexts: []*tw.WorkflowContext{
+			&tw.WorkflowContext{
+				WorkflowId:         "18eac798-fake-4250-bde9-812726eff314",
+				CurrentActionState: 0,
+			},
+		},
+	}
+	tinkWorkflowClient = func(hwID string) (tw.WorkflowContextList, error) {
+		//var result tw.WorkflowContextList
+		return wcl, nil
+	}
+	status, err := discoverActiveWorkflowForMac(hwID)
+	if err != nil {
+		t.Fatal("error occured while testing")
+	}
+	t.Logf("status of disovery action is %t", status)
+}
+
+func TestDiscoverActiveWorkflowForMacNotFound(t *testing.T) {
+	var hwID string = "0eba0bf8-fake-4b4a-ab9f-6ebe93b90a94"
```

Comment:
Same here, can you assert that the returned value is `status==false`?

Reply:
done